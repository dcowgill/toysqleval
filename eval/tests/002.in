create table test (s varchar);
insert into test values ('hello');
insert into test values (null);
insert into test values ('world');
select * from test;
