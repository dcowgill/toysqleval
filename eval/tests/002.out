OK
OK
OK
OK
 s      
---------
 "hello"
        
 "world"
